src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -122,8 +122,12 @@ public static void main(String[] args) throws IOException {
         boolean allValid = true;
         for (File fileToValidate : filesToValidate) {
             logger.info("Validating " + fileToValidate);
-            final Model dataToBeValidated = loadModel(Files.asCharSource(fileToValidate, Charsets.UTF_8)
-                    .openBufferedStream());
+            final Model dataToBeValidated = ModelFactory.createOntologyModel();
+            try {
+                loadModel(dataToBeValidated, Files.asCharSource(fileToValidate, Charsets.UTF_8));
+            } catch (Exception e) {
```

Comment:
If you're unable to load the file to be validated, do you want the validation to continue anyway? If so, we shouldn't catch the RuntimeException.

Reply:
I think at the time I was thinking that it should log the failure but continue the batch validations, but the original Kotlin terminates, so I changed this one to do the same.